packages/pouchdb-adapter-leveldb-core/src/index.js

```diff
@@ -562,11 +562,15 @@ function LevelPouch(opts, callback) {
     }
 
     function finish() {
-      if (api.auto_compaction) {
-        return autoCompact(complete);
-      } else {
-        compact(stemmedRevs, complete);
-      }
+      compact(stemmedRevs, function (error) {
+        if (error) {
+          complete(error);
```

Comment:
This probably needs an istanbul ignore, I dont like adding them but with node callbacks its the only choice really